### ISSUE TITLE
not all snvs/indels uploaded

### ISSUE THREAD

**Issue by terestahl** [Clinical-Genomics/scout#3478]
Hi
I would like if it is possible that for cases in which not all snvs/indels are uploaded to scout due to e.g. high mutational burden, e.g. case LI015 that this is indicated somewhere in scout. It is important that this is noted somewhere to avoid wrong interpretations
thanks

teresita

**Comment by northwestwitch** [comment]
Mmm this is quite hard to achieve if you want to implement it only for specific cases (the high mutational burden ones, for instance). For rare diseases, at least for CG Solna, it is the rule to only load a small set of SNVs/INDELs. 

An easier solution might be to just indicate how many variants were present in the original VCF used when loading the case

**Comment by terestahl** [comment]
Ok this is fine, then it could be indicated the total number of "filtered" snvs/indels, not the raw number, to be able to compare to the number of snvs/indels in other cases
Thanks

teresita

**Comment by vwirta** [comment]
Do you think it would be sufficient to address this issue by providing a TMB score?
We are working on calculating this, but absence of international agreed joint definition complicates this. 
We could deliver the TMB score as part of the Delivery Report, which will be available via Scout.

**Comment by terestahl** [comment]
Yes this is a good idea, as far it is calculated in the same way for all the samples and explained how is it done somewhere, good!
